api/src/main/java/ca/bc/gov/educ/api/batchgraduation/config/RestWebClient.java

```diff
@@ -39,6 +45,7 @@ public WebClient getBatchWebClient(OAuth2AuthorizedClientManager authorizedClien
         ServletOAuth2AuthorizedClientExchangeFilterFunction filter = new ServletOAuth2AuthorizedClientExchangeFilterFunction(authorizedClientManager);
         filter.setDefaultClientRegistrationId("batch-client");
         return WebClient.builder()
+                .filter(setRequestHeaders())
```

Comment:
Please add a log() function as per other apis. You can see how others use it here: https://github.com/bcgov/EDUC-GRAD-GRADUATION-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/graduation/config/RestWebClient.java#L77